scripts/jenkins-yoctobuild-build.sh

```diff
@@ -1,23 +1,94 @@
 #!/bin/bash
-shopt -s extglob
-set -x
-export PATH=$WORKSPACE/scripts:$WORKSPACE/bitbake/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
+set -e -x
 
-rm -rf "$WORKSPACE/build/tmp/"
+export PATH=/home/jenkins/workspace/yoctobuild/scripts:/home/jenkins/workspace/yoctobuild/bitbake/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
```

Comment:
Rather keep `$WORKSPACE` and do this after the `export` below. Then it's much easier and less error prone to change later.